### DISCUSSION TITLE
Can the FlipView's Index indicator be modified?

### DISCUSSION THREAD

**Discussion by dknippel** [MahApps/MahApps.Metro#4427] [Q&A] [answered]
I would really like to be able to modify the Index indicator from an ellipse to perhaps a thumbnail + index number.  Also, is there anyway to put it into a ScrollViewer for times when my underlying collection is quite large?

Basically I'm needing a "filmstrip" type of Index indicator.  Any suggestions?

**Comment by timunie** [comment, marked as answer]
@dknippel check orignal styles and override / modify them as needed. 

https://github.com/MahApps/MahApps.Metro/blob/221996ba181414de311c7550a774d83555631258/src/MahApps.Metro/Themes/FlipView.xaml#L50-L78

**Reply by dknippel** [reply]
Excellent, thank you for your response @timunie !
